firstvoices/backend/views/page_views.py

```diff
@@ -59,10 +65,83 @@
             site_page_slug_parameter,
         ],
     ),
+    create=extend_schema(
+        description=_("Add a page."),
+        request=SitePageDetailWriteSerializer,
+        responses={
+            201: OpenApiResponse(
+                description=doc_strings.success_201,
+                response=inline_serializer(
+                    name="InlinePageDetailSerializer",
+                    fields={
+                        "id": serializers.UUIDField(),
+                        "title": serializers.CharField(),
+                        "url": serializers.URLField(),
+                        "visibility": serializers.CharField(),
+                        "subtitle": serializers.CharField(),
+                        "slug": serializers.CharField(),
+                        "widgets": SiteWidgetDetailSerializer(many=True),
+                        "banner_image": ImageSerializer(),
+                        "banner_video": VideoSerializer(),
```

Comment:
Just noticed these. Is there a reason we can't just refer to the existing serializer here? Same question for the other inline serializers as well

Reply:
If the `SitePageDetailSerializer` is used here than the widgets field formats incorrectly since the `SitePageDetailSerializer` uses the `SiteWidgetListSerializer` for the widgets field (which correctly creates the needed model instances) instead of the `SiteWidgetDetailSerializer` (which is the needed serializer for the API docs display).

In other words it looks like this:
![Screenshot 2023-07-26 at 3 33 13 PM](https://github.com/First-Peoples-Cultural-Council/fv-be/assets/3804042/203d398b-a2e1-44ac-b561-e5ad501e63d3)

But should look like this:
![Screenshot 2023-07-26 at 3 33 43 PM](https://github.com/First-Peoples-Cultural-Council/fv-be/assets/3804042/f3d6ddfd-c9ec-445b-9020-fbc2e36f2447)